generators/generator-bot-enterprise-assistant/generators/app/templates/README.md

```diff
@@ -43,13 +127,13 @@ You must configure an authentication connection on your **skill bot** Azure Bot
 
 3. Add the required Microsoft Graph scopes to your Microsoft App:
     ```
-    az rest --method patch --url https://graph.microsoft.com/v1.0/applications/<objectId> --body "{ 'requiredResourceAccess': [{'resourceAppId': '00000003-0000-0000-c000-000000000000', 'resourceAccess': [ { 'type': 'Scope', 'id': 'ba47897c-39ec-4d83-8086-ee8256fa737d' }, { 'type': 'Scope', 'id': 'ff74d97f-43af-4b68-9f2a-b77ee6968c5d' },  { 'type': 'Scope', 'id': '1ec239c2-d7c9-4623-a91a-a9775856bb36' }, { 'type': 'Scope', 'id': 'b340eb25-3456-403f-be2f-af7a0d370277' }, { 'id': 'b89f9189-71a5-4e70-b041-9887f0bc7e4a', 'type': 'Scope' }, { 'id': 'a154be20-db9c-4678-8ab7-66f6cc099a59',	'type': 'Scope'	}, { 'id': '06da0dbc-49e2-44d2-8312-53f166ab848a', 'type': 'Scope' } ]} ]}"
+    az rest --method patch --url https://graph.microsoft.com/v1.0/applications/<objectId> --body "{ 'requiredResourceAccess': [{'resourceAppId': '00000003-0000-0000-c000-000000000000','resourceAccess': [{ 'id': 'b89f9189-71a5-4e70-b041-9887f0bc7e4a', 'type': 'Scope' }, { 'id': 'b340eb25-3456-403f-be2f-af7a0d370277',	'type': 'Scope'	}, { 'id': 'a154be20-db9c-4678-8ab7-66f6cc099a59',	'type': 'Scope'	}, { 'id': '06da0dbc-49e2-44d2-8312-53f166ab848a', 'type': 'Scope' }, { 'id': 'ff74d97f-43af-4b68-9f2a-b77ee6968c5d', 'type': 'Scope'	}, { 'id': 'ba47897c-39ec-4d83-8086-ee8256fa737d', 'type': 'Scope' } ]}	]}"
```

Comment:
What are these IDs? I assume these IDs are simply the identifier for the scopes. We may want to map them to the actual textual name.

Reply:
These are the IDs. Since I don't have a script file for them to run, I didn't want to add any overhead as far as an additional mapping method goes. Let me know if you have ideas for how to communicate that here

Reply:
I think having a simple table or bulleted list underneath works. 

For example:

> Below are mapping between each IDs mentioned above and their Permission Name in Microsoft Graph.
> - People.Read.All = b89f9189-71a5-4e70-b041-9887f0bc7e4a
> - User.ReadBasic.All = b340eb25-3456-403f-be2f-af7a0d370277
> - ...

Reply:
Lets hold off on this addition for now. I'll add it next week